shared/src/test/resources/vocabularies2/instances/example34.raml

```diff
@@ -1,11 +1,15 @@
 #%Test 1.34
-
 bs:
   B1:
     id: a
     b: 3.4
-
 a: uno
 links:
-  - http://test/a
-  - http://test/other
+  - http://test/simple
+  -
+    $id: http://test/simpleAlt
+  -
+    id: embed
+  -
+    id: embedOverwritten
+    $id: http://defined.by.me
```

Comment:
This last example looks strange to me. As defined in the dialect node `B` has `idTemplate: http://test/{id}`. The last element in this list defines both `$id: http://defined.by.me` and `id: embedOverwritten` and the resulting node ID in the JSON-LD is the concatenation of both `http://defined.by.me/embedOverwritten`. Shouldn't the $id facet (in the instance) override the idTemplate pattern and discard the `/{id}` concatenation? @antoniogarrote